sound/soc/sof/loader.c

```diff
@@ -154,6 +154,18 @@ static int ext_man_get_windows(struct snd_sof_dev *sdev,
 	return get_ext_windows(sdev, w_ipc);
 }
 
+static int ext_man_get_cc_info(struct snd_sof_dev *sdev,
+			       const struct ext_man_elem_header *hdr)
+{
+	const struct ext_man_cc_version *cc;
+	const struct sof_ipc_ext_data_hdr *cc_version;
+
+	cc = container_of(hdr, struct ext_man_cc_version, hdr);
+	cc_version = (const struct sof_ipc_ext_data_hdr *)&cc->cc_version;
+
+	return get_cc_info(sdev, cc_version);
```

Comment:
just do `return get_cc_info(sdev, &cc->cc_version);`

Reply:
`sof_ipc_cc_version *`  shouldn't cast to `sof_ipc_ext_data_hdr *' by default